database/dtb.ksy

```diff
@@ -0,0 +1,94 @@
+meta:
+  id: dtb
+  title: Flattened Devicetree Format
+  file-extension: dtb
+  license: CC0-1.0
+  endian: be
+doc: |
+  Also referred to as Devicetree Blob (DTB). It is a flat
+  binary encoding of data (primarily devicetree data, although
+  other data is possible as well).
+doc-ref:
+  - https://github.com/devicetree-org/devicetree-specification/releases/download/v0.3/devicetree-specification-v0.3.pdf
+  - https://elinux.org/images/f/f4/Elc2013_Fernandes.pdf
+seq:
+  - id: magic
+    contents: [0xd0, 0x0d, 0xfe, 0xed]
+  - id: total_size
+    type: u4
+  - id: structure_block_offset
+    type: u4
+  - id: strings_block_offset
+    type: u4
+  - id: memory_reservation_block_offset
+    type: u4
+  - id: version
+    type: u4
+  - id: last_compatible_version
+    type: u4
+  - id: boot_cpuid_phys
+    type: u4
+  - id: size_dt_strings
```

Comment:
the `id`s should be renamed: they go to the generated parser. Information about original ids should be preserved using `-orig-id`s.

Reply:
I renamed some to give them more useful names, some I have kept as I don't have a better name for them.